activeagent.gemspec

```diff
@@ -26,5 +26,5 @@ Gem::Specification.new do |spec|
   spec.add_dependency "activejob", ">= 7.2", "< 9.0"
 
   spec.add_dependency "rails", ">= 7.2", "< 9.0"
-  spec.add_development_dependency "jbuilder"
+  spec.add_dependency "jbuilder"
```

Comment:
Not everyone uses jbuilder, so I'd prefer keeping it a dev dependency for testing jbuilder without forcing it on everyone as a dependency.